Content.Shared/Zombies/ZombieComponent.cs

```diff
@@ -140,6 +140,12 @@ public sealed partial class ZombieComponent : Component, IAntagStatusIconCompone
     [DataField("greetSoundNotification")]
     public SoundSpecifier GreetSoundNotification = new SoundPathSpecifier("/Audio/Ambience/Antag/zombie_start.ogg");
 
+    /// <summary>
+    ///     Hit sound on zombie bite.
+    /// </summary>
+    [DataField("biteSound")]
+    public SoundSpecifier BiteSound = new SoundPathSpecifier("/Audio/Effects/bite.ogg");
+
```

Comment:
```suggestion
    /// <summary>
    ///     Hit sound on zombie bite.
    /// </summary>
    [DataField]
    public SoundSpecifier BiteSound = new SoundPathSpecifier("/Audio/Effects/bite.ogg");

```

Datafield strings are long obsolete.